app/controllers/application_controller.rb

```diff
@@ -987,8 +987,6 @@ def view_to_hash(view, fetch_data = false)
         celltext = nil
 
         case view.col_order[col_idx]
-        when 'db'
```

Comment:
How do you know that `db` is unused ?

Reply:
It's not present in `product/views/` under `cols` or `col_order`.